### PR TITLE
Updated to Node 18

### PR DESCRIPTION
**Pull request by bvitaliyg**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] Component `README.md`
+ [ ] Tests

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

**Comment by anambl** [comment]
Also, can you have a look if you haven't already for any references to v16 in guidelines, readmes etc?

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-lts/gallium
+v18
```
Comment (frugoman) [inline .nvmrc:1]:
```suggestion
lts/hydrogen
```

[the remark this answers is not in the document]
Comment (bvitaliyg) [reply .nvmrc]: CC @anambl

[the remark this answers is not in the document]
Comment (anambl) [reply .nvmrc]: It's not exactly the same, this ensures you're always on the lts version where as v18 just means you're on any 18.x.x version. Most likely they'll always be the same as I think node uses all even number versions as lts, but this is safer and plus it's what we use in all projects within the team and in the company too

**Pull request merged**